packages/wonder-blocks-testing/src/fixtures/fixtures.js

```diff
@@ -1,127 +1,78 @@
 // @flow
 import * as React from "react";
-import {getConfiguration} from "./setup.js";
-import {combineOptions} from "./combine-options.js";
+import {action} from "@storybook/addon-actions";
```

Comment:
This file is basically most of what was in `storybook.js` but modified to work with our new syntax.